system_files/usr/share/containers/storage.conf

```diff
@@ -0,0 +1,162 @@
+# This file is the configuration file for all tools
```

Comment:
How does this file differ from the default? Maybe include a comment at the top with a quick note of the differences

Reply:
The default breaks podman. I agree